sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -8,7 +8,7 @@
 
 import type { AbortSignalLike } from '@azure/abort-controller';
 import type { AccessToken } from '@azure/core-auth';
-import type { AzureLogger } from '@azure/logger';
```

Comment:
this losing its `type` feels suspicious

Reply:
It does, but there's nowhere that I removed `type` in front of the `AzureLogger` so I too am suspicious.